app/services/rapid_ready_for_decision/fast_track_pdf_generator.rb

```diff
@@ -24,13 +25,27 @@ def self.extract_patient_name(patient_info)
       [full_name, patient_info[:suffix]].reject(&:blank?).join ', '
     end
 
+    # TODO: move this into the data class
+    def self.flag_with_keyword(keywords, medication_struct)
+      keywords.any? { |keyword| medication_struct.to_s.include?(keyword) }
+    end
```

Comment:
You may have to split the keywords into 2 sets: medications and dosage instructions.
Also when you search for the keyword, ignore capitalizations.

Reply:
> When you search for the keyword, ignore capitalizations: `medication_struct.to_s.downcase` and downcase the `keywords` in `Constants`.
> 
> ~You may have to split the keywords into 2 sets: medications and dosage instructions.~ Nevermind, I think this is acceptable -- we'll have to loop back with Researchers to confirm.

Yeah, I thought about that, and I think ultimately you're right, it needs to allow for that kind of specificity. Your point is another argument for having this behavior moved into the `LighthouseMedicationRequestData` class, where it could add a `flagged` key to the `medication_struct` if it meets certain criteria... still, will get this wrapped up asap...

Reply:
You could build your own medication results struct that has the desired flag. You can start with something like this: https://github.com/department-of-veterans-affairs/vets-api/blob/90e350090d5afb98604e370a6d0cc732c6ab2e91/app/workers/rapid_ready_for_decision/form526_hypertension_job.rb#L25-L30

and then use it during PDF generation.
I would keep the code in the asthma processor/job since the meds are specific to asthma, and keep `LighthouseMedicationRequestData` general.

Reply:
@yoomlam yes! even better! thanks